libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -87,32 +69,63 @@ class PredictiveCacheController @JvmOverloads constructor(
     }
 
     /**
-     * Call when a new map instance is available. Only one map instance at a time is supported.
+     * Create cache controllers for a map instance.
+     * Call when a new map instance is available.
```

Comment:
Add the same docs about how the ID should be formatted, etc. The same as the class docs.